pkg/cli/options.go

```diff
@@ -44,53 +44,47 @@ func WithVersion(version string) Option {
 	}
 }
 
-// WithDefaultProjectVersion is an Option that sets the cli's default project version.
-// Setting an unknown version will result in an error.
-func WithDefaultProjectVersion(version config.Version) Option {
+// WithPlugins is an Option that sets the cli's plugins.
```

Comment:
Could we improve all comments docs over what is exported with examples indeed of its usage?